src/util/index.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  approveAndMergePullRequestMutation,
+  mergePullRequestMutation,
+} from '../graphql/mutations';
+/**
+ * This method returns the right GraphQl mutation depending on weather the
```

Comment:
I think we can simplify it and align with other repositories.

```suggestion
 * Returns the right GraphQl mutation depending on weather the
```